server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/DefaultCompactionFinalizer.java

```diff
@@ -53,28 +57,29 @@
 import org.apache.accumulo.core.metadata.schema.TabletsMetadata;
 import org.apache.accumulo.core.rpc.ThriftUtil;
 import org.apache.accumulo.core.rpc.clients.ThriftClientTypes;
+import org.apache.accumulo.core.spi.compaction.CompactionFinalizer;
 import org.apache.accumulo.core.tabletserver.thrift.TabletClientService;
 import org.apache.accumulo.core.trace.TraceUtil;
 import org.apache.accumulo.core.util.Timer;
 import org.apache.accumulo.core.util.threads.ThreadPools;
-import org.apache.accumulo.server.ServerContext;
 import org.apache.thrift.TException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class CompactionFinalizer {
+public class DefaultCompactionFinalizer implements CompactionFinalizer {
```

Comment:
"Default" is a bad name for the implementation of a configurable class. The name should reflect the behavior or strategy it implements, not its role as the default in the configuration, because that can change.

Imagine the case where a new implementation is added and replaces this one as the default. Now, a class *not* named "Default" is now the default, and the class named "Default" is *not* the default. That is inherently confusing.

Alternate names might be "SimpleCompactionFinalizer" or "BasicCompactionFinalizer" if the implementation employs a basic strategy that doesn't have a specific name. If it employs a specific strategy, like "Random" or "Periodic" or "Logging" or "Delegating", then that can be part of the name instead of "Default".